test-foundry/TestHeapOrdering.t.sol

```diff
@@ -503,4 +502,14 @@ contract TestHeapOrdering is DSTest {
 
         assertEq(heap.accounts[1].value, 25);
     }
+
+    function testOverflowNewValue() public {
+        hevm.expectRevert("SafeCast: value doesn't fit in 96 bits");
+        update(accounts[0], 0, uint256(type(uint128).max));
```

Comment:
why not `type(uint128).max + 1`?

Reply:
It would revert with the change you propose. I can change it to `uint256(type(uint96).max)+1` but it is longer

Reply:
It is ok like this IMO

Reply:
ok np